Gui/typesystem_natronGui.xml

```diff
@@ -19,8 +19,14 @@
    - ***** END LICENSE BLOCK ***** -->
 <typesystem package="NatronGui">
 
+    <inject-code class="native" position="beginning">
+        NATRON_NAMESPACE_USING NATRON_PYTHON_NAMESPACE_USING
+    </inject-code>
+
     <!--Load QtGui typesystem-->
     <load-typesystem name="typesystem_gui.xml" generate="no" />
+    <!--Load QtWidgets typesystem-->
```

Comment:
Yes, It'll do, although I _**think**_ we can workaround this by creating a `typesystem_widgets.xml` file like, e.g.

```xml
<?xml version="1.0"?>
<typesystem package="SnakeOil">
</typesystem>
```

and then adding a `--typesystem-paths` argument that points to its directory.

Reply:
so did you test if it still works with shiboken? I'd really like to have both shiboken and shiboken2 in the main branch. If that means having two typesystem XML files, that's ok.

Reply:
I've run a test with old `shiboken` and the `SnakeOil` package with the empty typesystem and it worked. Both `shiboken2` and `shiboken6` can do it as well. Keep in mind that this file must reside in a directory that's different from `Engine` and `Gui` so that the package can be generated by later versions, my suggestion is to use `tools` or a subpath of it.

Reply:
Nice!
why not a directory named Shiboken instead, so that it's clear that it's part of the sources and not a separate tool?
Can you make the change?

Reply:
I can't decide whether to create a `tools/Shiboken1Compat` or a `Shiboken` dir. If you tell which one is better I'll upload the patch really quick.

Reply:
`Shiboken`.
`tools/*` is all the stuff that is not Natron source code (CI/CD, building third party libraries and tools, etc).